components/scream/cmake/ScreamUtils.cmake

```diff
@@ -3,11 +3,15 @@
 #    - target_srcs: a list of src files for the executable
 #      Note: no need to include catch_main; this macro will add it
 #    - scream_libs: a list of scream libraries needed by the executable
-#    - config_defines: a list of additional defines for the compiler
-#    - num_mpi_ranks: the number of mpi ranks for the test
-#    - config_defines: specific define directives to be passed to the compiler
+#    - config_defs (optional): a list of additional defines for the compiler, default is nothing
+#    - mpi_ranks (optional): the number of mpi ranks for the test, if 2 values, it's a range, if 3, it's a range plus increment. default is np=1
+#    - threads (optional): the number of threads for the test, if 2 values, it's a range, if 3, it's a range plus an increment. default is 1 thread
```

Comment:
If I understand correctly, if given a range of ranks/threads, you create one test per combination, right? I would add a comment stating this, so it's clear from the comments what the ranges yield.